### PR TITLE
Document basic usage

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- added more content to overview section
- summarized basic usage scenario (in two variants)
- cleaned up various md files locations

**Related issue(s)**
#82

### CODE REVIEW

**Review by Cortey — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,7 +7,12 @@
 
 ## Overview
 
-Docker Registry Operator allows deploying the [Docker Registry](docs/user/README.md) component in the Kyma cluster in compatibility with [Lifecycle Manager](https://github.com/kyma-project/lifecycle-manager).
+Docker Registry module aims to provide a lightweight, open-source Docker registry for storing and distributing container images in kubernetes environment. It could be used in self-contained development clusters or testing environments allowing you to test deployments in an isolated system.
```
Comment (IwonaLanger) [inline README.md:10]:
```suggestion
Docker Registry module aims to provide a lightweight, open-source Docker registry for storing and distributing container images in the Kubernetes environment. You can use it in self-contained development clusters or testing environments, which allows you to test deployments in an isolated system. 
```

File: README.md [hunk 2]
```diff
@@ -7,7 +7,12 @@
 
 ## Overview
 
-Docker Registry Operator allows deploying the [Docker Registry](docs/user/README.md) component in the Kyma cluster in compatibility with [Lifecycle Manager](https://github.com/kyma-project/lifecycle-manager).
+Docker Registry module aims to provide a lightweight, open-source Docker registry for storing and distributing container images in kubernetes environment. It could be used in self-contained development clusters or testing environments allowing you to test deployments in an isolated system. 
+
+> [!WARNING]  
+> Do not use on production clusters, where a full-fledged, highly-available, production-grade registry is necessary.
```
Comment (IwonaLanger) [inline README.md:13]:
```suggestion
> Do not use Docker Registry in production clusters, where a full-fledged, highly-available, production-grade registry is necessary.
```

File: README.md [hunk 3]
```diff
@@ -7,7 +7,12 @@
...
+
+Docker Registry module comes with a dedicated kubernetes operator for efortless installation and upgrades. It manages the lifecycle of the Docker Registry resources according to the desired specification expressed via dedicated `Dockerregistry` Custom Resource. Read more about configuration options in the [user documentation](./docs/user).
```
Comment (IwonaLanger) [inline README.md:15]:
```suggestion
The Docker Registry module comes with a dedicated Kubernetes operator for effortless installation and upgrades. It manages the lifecycle of the Docker Registry resources according to the desired specification expressed through a dedicated `Dockerregistry` custom resource (CR). Read more about configuration options in the [user documentation](./docs/user). 
```

File: README.md [hunk 4]
```diff
@@ -29,6 +34,13 @@ kubectl apply -f https://github.com/kyma-project/docker-registry/releases/latest
 kubectl apply -f https://github.com/kyma-project/docker-registry/releases/latest/download/default-dockerregistry-cr.yaml
 ```
 
+## Usage
+
+Having installed Docker Registry module, you can start pushing images to it via [kyma CLI (v3)](https://github.com/kyma-project/cli?tab=readme-ov-file#install) or, if you decide to expose the registry externally, using Docker CLI.
```
Comment (IwonaLanger) [inline README.md:39]:
```suggestion
Having installed the Docker Registry module, you can start pushing images to it using [kyma CLI (v3)](https://github.com/kyma-project/cli?tab=readme-ov-file#install) or, if you decide to expose the registry externally, using Docker CLI.
```

File: README.md [hunk 5]
```diff
@@ -29,6 +34,13 @@ kubectl apply -f https://github.com/kyma-project/docker-registry/releases/latest
...
+Please find both variants described in the dedicated tutorials:
+ - [How to use non exposed registry](docs/user/tutorials/01-10-simple-usage.md)
```
Comment (IwonaLanger) [inline README.md:41]:
```suggestion
 - [How to Use Non Exposed Registry](docs/user/tutorials/01-10-simple-usage.md)
```

File: docs/user/tutorials/01-20-expose-registry.md [hunk 6]
```diff
@@ -18,68 +18,55 @@ This tutorial shows how you can expose the registry to the outside of the cluste
     >[!NOTE] 
     > You can find your cluster address in the `kyma-system/kyma-gateway` gateway resource.
 
-1. Expose the registry service using a VirtualService CR based on the `kyma-gateway` gateway in the `kyma-system` namespace:
+1. Expose the registry service by changing the `spec.externalAccess.enabled` flag to true. Optionally, you could also change the host name:
```
Comment (IwonaLanger) [inline docs/user/tutorials/01-20-expose-registry.md:21]:
```suggestion
1. Expose the registry service by changing the **spec.externalAccess.enabled** flag to `true`. Optionally, you can also change the host name:
```

File: README.md [hunk 7]
```diff
@@ -29,6 +34,13 @@ kubectl apply -f https://github.com/kyma-project/docker-registry/releases/latest
 kubectl apply -f https://github.com/kyma-project/docker-registry/releases/latest/download/default-dockerregistry-cr.yaml
```
Comment (IwonaLanger) [inline README.md:34]: Please indent the code blocks in this section to make the numbered points better visible.

File: docs/user/tutorials/01-20-expose-registry.md [hunk 8]
```diff
@@ -18,68 +18,55 @@ This tutorial shows how you can expose the registry to the outside of the cluste
...
-                host: dockerregistry.kyma-system.svc.cluster.local
-                port:
-                    number: 5000
+      externalAccess:
+        enabled: true
+        hostPrefix: my-registry
     EOF
     ```
+   
+   Once DockerRegistry CR becomes `Ready`, you will see a secret name that shall be used as `ImagePullSecret` when scheduling workloads in the cluster.
```
Comment (IwonaLanger) [inline docs/user/tutorials/01-20-expose-registry.md:37]:
```suggestion
   Once the DockerRegistry CR becomes `Ready`, you see a Secret name that is used as `ImagePullSecret` when scheduling workloads in the cluster.
```

**Pull request merged**